styles.css

```diff
@@ -1,47 +1,60 @@
 * {
   box-sizing: border-box;
+  font-family: poppins, sans-serif;
+  margin: 0;
+  padding: 0;
 }
 
-body {
-  display: flex;
-  font-family: poppins, sans-serif;
-  flex-direction: column;
-  margin: 0 auto;
-  width: 375px;
+main {
+  max-width: 400px;
+  margin: auto;
+  background-color: #dfe1e6;
```

Comment:
- [ ] [ OPTIONAL ] Though I'm marking this request as optional, I **HIGHLY** recommend you to implement it because the next reviewer might ask you to implement it.
The background color of your **main** tag should have a lighter shade of gray. The right code color to use is **#f7f7f9**
### Note that it is always good practice to pay attention, even to the littlest detail 🕵🏽 🔍